pkg/algorithm/nodeselect/v1alpha1/select_node.go

```diff
@@ -371,6 +371,11 @@ func (ac *Config) ClaimBlockDevice(nodeBDs *nodeBlockDevice, spc *apis.StoragePo
 			}
 			capacity := volume.ByteCount(bdObj.Spec.Capacity.Storage)
 
+			// WithOwnerReference method accepts an empty interface to enable
+			// setting of OwnerReference to other types of objects
+			var spc_owner interface{}
```

Comment:
Instead of making this `interface{}`, the [following](https://github.com/openebs/maya/pull/1795/files#diff-6e2062236a6ff2f9de548897c2711c30a29e107feb6b984253afb135a7985067R290) in builder can be used here.